includes/wizards/newspack/class-newspack-settings.php

```diff
@@ -90,25 +90,46 @@ public function get_local_data() {
 					],
 				],
 			],
-			'social'            => [
+			'social'           => [
 				'label' => __( 'Social', 'newspack-plugin' ),
 			],
-			'syndication'       => [
+			'syndication'      => [
 				'label' => __( 'Syndication', 'newspack-plugin' ),
 			],
-			'seo'               => [
+			'seo'              => [
 				'label' => __( 'SEO', 'newspack-plugin' ),
 			],
-			'theme-and-brand'   => [
+			'theme-and-brand'  => [
 				'label' => __( 'Theme and Brand', 'newspack-plugin' ),
 			],
-			'display-settings'  => [
+			'display-settings' => [
 				'label' => __( 'Display Settings', 'newspack-plugin' ),
 			],
-			'additional-brands' => [
-				'label' => __( 'Additional Brands', 'newspack-plugin' ),
-			],
 		];
+		if ( is_plugin_active( 'newspack-multibranded-site/newspack-multibranded-site.php' ) ) {
```

Comment:
This check is a bit fragile. There are cases in which the plugin is installed with a different path, sometimes for testing or other reasons. WDYT of changing this to `defined( 'NEWSPACK_MULTIBRANDED_SITE_PLUGIN_FILE' )`?

Reply:
Updated c256a232f032dc023096f78544691b1a0acae821